modules/seedtagBidAdapter.js

```diff
@@ -114,7 +128,10 @@ function buildBid(seedtagBid) {
     netRevenue: true,
     mediaType: mediaType,
     ttl: seedtagBid.ttl,
-    nurl: seedtagBid.nurl
+    nurl: seedtagBid.nurl,
+    meta: {
+      advertiserDomains: seedtagBid && seedtagBid.adomain && seedtagBid.adomain.length > 0 ? seedtagBid.adomain : []
```

Comment:
i think should probably be `seedtagBid.adomain && seedtagBid.adomain.length > 0 ? seedtagBid.adomain : []`